# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/base/nsFrameLoader.cpp
# Commit: 5d11d295d7d4
# Full Hash: 5d11d295d7d4c0431ea0a772a763a876f4fd0ef5
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2019-05-28 21:44:47
# Description:
#   Bug 1554073 - Use GetContentParent in TryRemoteBrowser to avoid null-deref if PBrowser actor is destroyed. r=nika
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D32547
# ==============================================================================

diff -r c30c24753844 -r 5d11d295d7d4 dom/base/nsFrameLoader.cpp
--- a/dom/base/nsFrameLoader.cpp	Tue May 28 13:37:08 2019 +0000
+++ b/dom/base/nsFrameLoader.cpp	Mon May 27 17:31:24 2019 +0000
@@ -2605,7 +2605,7 @@
   RefPtr<ContentParent> openerContentParent;
   RefPtr<BrowserParent> sameTabGroupAs;
   if (auto* host = BrowserHost::GetFrom(parentDocShell->GetOpener())) {
-    openerContentParent = host->GetActor()->Manager();
+    openerContentParent = host->GetContentParent();
   }
 
   // <iframe mozbrowser> gets to skip these checks.
